### ISSUE TITLE
Simplify download page

### ISSUE THREAD

**Issue by smoyth** [opendatakit/website#91]
Right now it comes across as three articles or blog posts.

Suggesting a simplified, spacious two-column layout, maybe without the images, with the two questions ("Looking for ..", "Have some..") prominently featured.

Perhaps like this:

![image](https://user-images.githubusercontent.com/702320/40386445-bd715e26-5dd7-11e8-9e32-9afae735cc11.png)

It might be nice to reduce the amount of text under each. I was thinking if each of the key tools (Collect, Aggregate, Tables, Survey, etc.) had their own visual identity, even just an icon, we could feature those brands right under the `h2`s. Do we have icons for all the tools?

I also am not sure about the 'Help me choose' section. There is a lot of text and then some of the comparison items seem a bit arbitrary. I wonder if we shouldn't just start with something simpler first.

**Comment by wbrunette** [comment]
I agree with @smoyth that we probably do not need the third column. 

I also like the idea of visual identity of the tools using their icons; however not sure it has to be done before the launch could be done later but would be happy if it was.

A comparison table between ODK 1 and ODK 2 has been requested multiple times. Additions to the table or other modifications are welcome but it has been requested at the convening, on the forum, in email, etc.

**Comment by yanokwa** [comment]
I think it's going to be hard to get a visual identity done before launch, but we can definitely file a separate issue. 

As to comparison table, I do think this will be contentious. What would be easy to do also is to add bit more text to the different columns about what each thing does and don't compare. For me, the important things are the stage of technology, adoption, and one or two tent-pole features should be enough to get people started.

**Comment by smoyth** [comment]
I like Yaw's suggestion. I think perhaps it's a less competitive/adversarial way of giving users what they need in terms of judging the two suites. There could even be more than one or two features, but this way there wouldn't be the "we have it and they don't" kind of messaging. Instead it would be "here are our strengths" and "here are our strengths" messaging, which I think is more appropriate.

**Comment by yanokwa** [comment]
I'll take a stab at reducing this page into two columns and send in a PR for review.

**Comment by yanokwa** [comment]
@opendatakit-bot Don't make me unplug you...

**Comment by smoyth** [comment]
lol
